src/components/editor/property/InputLookupQA.jsx

```diff
@@ -1,37 +1,38 @@
 // Copyright 2019 Stanford University see LICENSE for license
 
-import React from 'react'
+import React, { useState } from 'react'
 import {
   Menu, MenuItem, Typeahead, asyncContainer, Token,
 } from 'react-bootstrap-typeahead'
 import { getOptionLabel } from 'react-bootstrap-typeahead/lib/utils'
-
 import PropTypes from 'prop-types'
 import SinopiaPropTypes from 'SinopiaPropTypes'
+import shortid from 'shortid'
 import { bindActionCreators } from 'redux'
 import { connect } from 'react-redux'
 import {
   itemsForProperty, getDisplayValidations, getPropertyTemplate, findErrors,
 } from 'selectors/resourceSelectors'
 import { changeSelections } from 'actions/index'
-import search from 'actionCreators/qa'
+import { getSearchResults } from 'actionCreators/qa'
 import { isValidURI } from 'Utilities'
 import { booleanPropertyFromTemplate } from 'utilities/propertyTemplates'
 import _ from 'lodash'
 import RenderLookupContext from './RenderLookupContext'
 
 const AsyncTypeahead = asyncContainer(Typeahead)
 
-// Render menu function to be used by typeahead
+
 export const renderMenuFunc = (results, menuProps) => {
   const items = []
-  let menuItemIndex = 0
+  let authURI
+  let authLabel
 
   /*
-   * Returning results per each promise
+   * Returning results
    * If error is returned, it will be used to display for that source
    */
-  results.forEach((result, _i, list) => {
+  results.forEach((result, _i) => {
```

Comment:
If these variables are being used, I don't think they should be `_`.

Reply:
Done.